examples/amp-story/attachment.html

```diff
@@ -51,7 +51,14 @@
             <span data-text-background-color="#00A3DB">Relentlessly pursues moth annoy owner until he gives you food say meow repeatedly until belly rubs, feels good drool that box?</span>
           </p>
         </amp-story-grid-layer>
-        <amp-story-page-attachment data-title="Lorem ipsum" layout="nodisplay">
+        <amp-story-page-attachment
+          data-title="Lorem ipsum"
+          layout="nodisplay"
+          cta-text="Call To Action"
+          cta-img="/extensions/amp-story/img/sweater.jpg"
+          cta-img-2="/extensions/amp-story/img/boot.jpg"
```

Comment:
a11y:
Publishers need a way to specify alt text for these images.
Each image needs a `-alt` attribute, like `cta-image-alt` and `cta-img-2-alt`.